easypy/concurrency.py

```diff
@@ -936,6 +954,10 @@ def _logged_func(self):
         except Exception as exc:
             _logger.silent_exception("Exception in thread running %s (traceback can be found in debug-level logs)", self.func)
             self.exc = exc
+            try:
+                exc.timestamp = time.time()
+            except Exception:
```

Comment:
What kind of `Exception` are we expecting here?

Reply:
if exc happens to have __slots__ or some __setattribute__ or a timestamp readonly property it might not allow arbitrary attribute assignment.

Reply:
And we don't want a more specific exception? Or we just don't want to take the chance an exception logging related code will fail?